contexts/generic

```diff
@@ -105,7 +105,7 @@
             },
             "addProperties": {
                 "@id": "reproschema:addProperties",
-                "@type": "reproschema:AddProperties",
+                "@type": "reproschema:AddProperty",
```

Comment:
I think you have renamed this as `AdditionalProperties`

Reply:
should be `AdditionalProperty`